code/datums/diseases/transformation.dm

```diff
@@ -83,6 +83,8 @@
 		message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(affected_mob)]) to replace a jobbanned player.")
 		affected_mob.ghostize(0)
 		affected_mob.key = C.key
+		if(C.client?.holder && (CONFIG_GET(flag/auto_deadmin_antagonists) || C.client.prefs?.toggles & DEADMIN_ANTAGONIST))

```

Comment:
Is there any reason these aren't in mind transfer or mob/Login ? Unless you don't want this to happen outside of roundstart, eg admin getting antag later in the round.

Reply:
Oversight.